packages/zk-transfer-web/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@types/jest": "^24.0.15",
+    "@types/lodash": "^4.14.149",
```

Comment:
if it's just for one function, let's not import all of `lodash` and its types and do it directly in javascript (see other comment below)

Reply:
Like this: `import isUndefined from 'lodash/isUndefined';`?

Reply:
See other change below. Don't use `isUndefined` or `lodash`, just compare the value directly, thanks.

Reply:
Okay!